plumgrid_playbooks/roles/plumgrid_gateway/files/update_gw_ifc.sh

```diff
@@ -51,7 +51,7 @@ while [ $# -gt 0 ]; do
 done
 
 if [ "$update_option" = "add" ]; then
-  if grep -Fq "$interface" /var/lib/libvirt/filesystems/plumgrid/var/run/plumgrid/lxc/ifc_list_gateway
+  if grep -Fq "$interface" /var/run/plumgrid/ifc_list_gateway
```

Comment:
You're hardcoding this to the docker relative paths. Why not determine directory paths dynamically like before?